Aptoide/aptoide-client-v8
# Hack/[AS-104] endless scroll on curation list

**What does this PR do?**

   Adds the possibility to do an endless scroll on the curation list. (Needs to be mocked as API is still not ready to deliver more than 1)

**Database changed?**

 No

**Where should the reviewer start?**

EditorialiListFragment.java

**How should this be manually tested?**

Mock charles (change total and add curation cards), scroll to the bottom, repeat the mock, check if it appears new cards

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/AS-104


**Code Review Checklist**
